src/functions.jl

```diff
@@ -0,0 +1,47 @@
+using DataFrames
+"""
+    npt_running_average(x_val, y_val, n = 5, x_name = "x", y_name = "y")
+    
+# Defines a running average function
```

Comment:
This documentation string does not follow [Julia's conventions for docstrings](https://docs.julialang.org/en/v1/manual/documentation/).